### PR TITLE
Development grouplevel schemas

### PR DESCRIPTION
**Pull request by HarmdR**
Hi Lazlo,
Please check the changes for adding schema-id on group level when adding a new group.

### CODE REVIEW

File: group_manager/static/group_manager/js/group_manager.js [hunk 1]
```diff
@@ -897,6 +903,7 @@ $(function() {
 
             $(sel).filter('.selectify-category').each(function() {
                 var $el = $(this);
+                console.log('selectify category');
```
Comment (lwesterhof) [inline group_manager/static/group_manager/js/group_manager.js]: Leftover debug statement

File: group_manager/static/group_manager/js/group_manager.js [hunk 2]
```diff
@@ -1005,7 +1017,7 @@ $(function() {
 
             $(sel).filter('.selectify-subcategory').each(function() {
                 var $el = $(this);
-
+                console.log('selectify subcat')
```
Comment (lwesterhof) [inline group_manager/static/group_manager/js/group_manager.js]: Leftover debug statement

File: group_manager/group_manager.py [hunk 3]
```diff
@@ -21,8 +21,12 @@ def index() -> Response:
     user_type = response['data']['user_type']
     user_zone = response['data']['user_zone']
 
+    response = api.call('get_schemas', data={})
```
Comment (lwesterhof) [inline group_manager/group_manager.py]: Is this API call still needed? Looks like the schemas are retrieved by select2.

**Review by lwesterhof — CHANGES_REQUESTED** [review]

File: group_manager/static/group_manager/js/group_manager.js (hunk 1 above)
Comment (HarmdR) [reply group_manager/static/group_manager/js/group_manager.js]: fixed

File: group_manager/static/group_manager/js/group_manager.js (hunk 2 above)
Comment (HarmdR) [reply group_manager/static/group_manager/js/group_manager.js]: fixed

File: group_manager/group_manager.py (hunk 3 above)
Comment (HarmdR) [reply group_manager/group_manager.py]: This is the initial list for schema-validation purposes when a new group is added.
This list could be derived from the select2 but this select2 is not nessecarily used at that point.

**Pull request merged**